node/src/BaseClient.ts

```diff
@@ -1005,10 +1007,15 @@ export class BaseClient {
                       callbackIdx,
                       clusterScan: command,
                   })
-                : command_request.CommandRequest.create({
-                      callbackIdx,
-                      scriptInvocation: command,
-                  });
+                : command instanceof command_request.ResetConnectionPassword
```

Comment:
Can you use switch-case or if-else instead?

Reply:
We are taking it into another issue, and we will deconstruct all of it, the issue is already open, see my comments.